cdap-app-templates/cdap-program-report/src/main/java/co/cask/cdap/report/ReportGenerationSpark.java

```diff
@@ -99,31 +100,40 @@ public void run(JavaSparkExecutionContext sec) throws Exception {
   public static final class ReportSparkHandler extends AbstractSparkHttpServiceHandler {
 
     private static final Logger LOG = LoggerFactory.getLogger(ReportSparkHandler.class);
-    private static final Type REPORT_GENERATION_REQUEST_TYPE = new TypeToken<ReportGenerationRequest>() {
-    }.getType();
+    private static final Type REPORT_GENERATION_REQUEST_TYPE = new TypeToken<ReportGenerationRequest>() { }.getType();
     private static final String DEFAULT_LIMIT = "10000";
-    private SparkSession sparkSession;
+    private static final String READ_LIMIT = "readLimit";
+    private static final String START_FILE = "_START";
+    private static final String FAILURE_FILE = "_FAILURE";
+    private static final AtomicBoolean EXECUTOR_SHUTDOWN = new AtomicBoolean();
 
-    public static final String READ_LIMIT = "readLimit";
-    public static final String THREAD_POOL_SIZE = "poolSize";
-    public static final String START_FILE = "_START";
     public static final String REPORT_DIR = "reports";
     public static final String COUNT_FILE = "COUNT";
     public static final String SUCCESS_FILE = "_SUCCESS";
```

Comment:
are these ones used outside this class too ? if so, where ? and can they be moved to constants ?